src/Mono.Android/Mono.Android.csproj

```diff
@@ -373,4 +374,23 @@
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
+  <PropertyGroup>
+    <ApiCompatibilityDir>../../tests/api-compatibility</ApiCompatibilityDir>
+  </PropertyGroup>
+  <ItemGroup>
+    <ApiCompatibilityFiles Include="$(ApiCompatibilityDir)/acceptable-breakages-v*.txt" />
+    <ApiCompatibilityFiles Include="$(ApiCompatibilityDir)/reference/*.*" />
+  </ItemGroup>
+  <Target Name="AfterBuild" Inputs="$(TargetPath);@(ApiCompatibilityFiles)" Outputs="$(IntermediateOutputPath)CheckApiCompatibility.stamp">
```

Comment:
Targets should go into `Mono.Android.targets`, because some IDEs (*cough* Visual Studio for Mac *cough*) will *rewrite and re-format* `.csproj` files in certain circumstances, e.g. adding a new project to the `.sln`.  This is annoying and makes for extra work (e.g. reverting the change to the `.csproj`).

Reply:
`Inputs` and `Outputs` should also go on separate lines.

Reply:
I wasn't aware of this hiccup of VSMac, thanks for pointing it out.
All Fixed!